client/core/lib/Client.ts

```diff
@@ -30,14 +30,13 @@ interface FailedRegisterPolicy {
 }
 
 export interface InitOptions {
-  prefix?: string;
   url: string;
-  cache: Storage;
+  cache: StorageController | Storage | { storage: Storage; prefix?: string };
```

Comment:
Can we create a type for `{ storage: Storage; prefix?: string }`? we can call it CacheInitOptions...